app/components/custom_fields/details_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<%=
+  component_wrapper do
+    flex_layout do |content|
+      content.with_row(mb: 3) do
+        render Primer::Alpha::Banner.new(scheme: :default, icon: :info, dismiss_scheme: :hide) do
```

Comment:
```suggestion
        render Primer::Alpha::Banner.new(scheme: :default, icon: :info, dismiss_scheme: :none) do
```

🟢 Seems a sticky banner is more fitting in this case?

Reply:
This is explicitly sketched in the wire frames, should I ask Marc again?

Reply:
It might be a case of simple copy paste- it's a tiny thing but would be good to clarify 👍🏾